core/templates/components/main-nav/main-nav.twig

```diff
@@ -0,0 +1,388 @@
+
+  {% if has_search == true %}
+<nav {{ attributes }} id="su-main-nav" class="su-main-nav su-main-nav--mobile-search {{ modifier_class }}" aria-label="Main">
+  {% else %}
+<nav {{ attributes }} id="su-main-nav" class="su-main-nav {{ modifier_class }}" aria-label="Main">
+  {% endif %}
+  <button id="su-main-nav__toggle" class="su-main-nav__toggle {{ button_modifier_class }}" aria-expanded="false">Menu</button>
+  <ul class="su-main-nav__menu-lv1">
+    <li class="su-main-nav__item-lv1">
+      <a href="https://stanford.edu">Home</a>
+    </li>
+    <li class="su-main-nav__item-lv1 su-main-nav__item-lv1--parent su-main-nav__item-lv1--current">
+      <a href="https://stanford.edu" aria-expanded="false">Admissions &amp; Aid</a>
+      <ul class="su-main-nav__menu-lv2">
+        <li class="su-main-nav__item-lv2">
```

Comment:
This seems like a place where we might be able to reduce the number of css classes and write css selectors as `su-main-nav__menu-lvl2 > li` instead.